modular_skyrat/master_files/code/modules/clothing/under/akula_jobs.dm

```diff
@@ -5,28 +5,41 @@
 /// The proper layer to render the tail overlays onto
 #define TAIL_OVERLAY_LAYER 5.9
 
-/// A proc to be used in `equipped()` for all akula clothing which has the 'special tech' to keep their wearers slippery
-/obj/item/clothing/proc/apply_wetsuit_status_effect(mob/living/carbon/human/user)
+/datum/component/wetsuit
+	dupe_mode = COMPONENT_DUPE_UNIQUE
+
+/datum/component/wetsuit/RegisterWithParent()
+	. = ..()
+	RegisterSignal(parent, COMSIG_ITEM_EQUIPPED, PROC_REF(apply_wetsuit_status_effect))
+	RegisterSignal(parent, COMSIG_ITEM_DROPPED, PROC_REF(remove_wetsuit_status_effect))
+
+/datum/component/wetsuit/UnregisterFromParent()
+	. = ..()
+	UnregisterSignal(parent, list(
+		COMSIG_ITEM_EQUIPPED,
+		COMSIG_ITEM_DROPPED,
+	))
+
+/// A proc for all akula clothing which has the 'special tech' to keep their wearers slippery
+/datum/component/wetsuit/proc/apply_wetsuit_status_effect(obj/item/source, mob/living/user, slot)
+	if(slot == ITEM_SLOT_HANDS)
+		return FALSE
```

Comment:
This component should be in a component file, not in some random clothing file, please.